cypress/e2e/calendar/delete-existing-calendar.cy.js

```diff
@@ -0,0 +1,62 @@
+import {
+  MultiColumnListCell, Modal, Pane, Button, Link
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+
+describe('Delete existing calendars', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+      cy.openCalendarSettings();
+    });
+
+    cy.openCalendarSettings();
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+  });
+
+
+  it('deletes a single existing calendar', () => {
```

Comment:
it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {